common.h

```diff
@@ -209,18 +209,26 @@ typedef enum {
         REDIS_PROCESS_RESPONSE_CLOSURE(resp_func, ctx) \
     }
 
+/* Case insensitive compare against compile-time static string */
+#define REDIS_STRICMP_STATIC(s, len, sstr) \
+    (len == sizeof(sstr) - 1 && !strncasecmp(s, sstr, len))
+
+/* Test if a zval is a string and (case insensitive) matches a static string */
+#define ZVAL_STRICMP_STATIC(zv, sstr) \
+    REDIS_STRICMP_STATIC(Z_STRVAL_P(zv), Z_STRLEN_P(zv), sstr)
+
+/* Case insensitive compare of a zend_string to a static string */
+#define ZSTR_CMP_STATIC(zs, sstr) \
```

Comment:
Shouldn't this be `ZSTR_ICMP_STATIC`?

Reply:
Good point.  Maybe `ZSTR_STRICMP_STATIC` just to be totally OCD about it.